sklearn/utils/multiclass.py

```diff
@@ -350,6 +350,23 @@ def _check_partial_fit_first_call(clf, classes=None):
     return False
 
 
+def _pfit_validate_y_against_classes(y, classes):
+    """
+    Checks if all target labels of current batch np.unique(y) are contained in
+    the classes argument passed during the first partial_fit call.
+    If not, a ValueError is raised.
+    """
+    
+    unique_y = np.unique(y)
+    unique_y_in_classes = in1d(unique_y, classes)
+    
+    if not np.all(unique_y_in_classes):
+         raise ValueError(
+                 "The target label(s) %s in y do not exist in the "
+                 "initial classes %s" % (unique_y[~unique_y_in_classes],
+                                         classes))
+
+
```

Comment:
Does this function name look okay? also is the addition of this helper justified?